File: core/src/org/rexcellentgames/burningknight/entity/item/reference/StarCannon.java
package org.rexcellentgames.burningknight.entity.item.reference;

import org.rexcellentgames.burningknight.entity.item.weapon.gun.Gun;
import org.rexcellentgames.burningknight.entity.item.weapon.gun.bullet.Star;

public class StarCannon extends Gun {
  {

    damage = 4;
    ammo = Star.class;
    setAccuracy(1f);
    penetrates = true;  }
}

File: core/src/org/rexcellentgames/burningknight/entity/item/reference/StarCannon.kt
package org.rexcellentgames.burningknight.entity.item.reference

import org.rexcellentgames.burningknight.entity.item.weapon.gun.Gun
import org.rexcellentgames.burningknight.entity.item.weapon.gun.bullet.Star

class StarCannon : Gun() {
  init {
    damage = 4
    ammo = Star::class.java
    accuracy = 1f
    penetrates = true
  }
}